systems/plants/@RigidBodyManipulator/getCMMdA.m

```diff
@@ -1,8 +1,14 @@
-function [A,dA] = getCMMdA(model, kinsol, robotnum, in_terms_of_qdot)
-% returns the centroidal momentum matrix as described in Orin & Goswami 2008
+function [A, dA] = getCMMdA(robot, kinsol, robotnum, in_terms_of_qdot)
```

Comment:
seems like we should consider renaming this to simply `getCMM` now? possibly even catching the old input (qd instead of robotnum) and throwing a deprecated-style warning?